.github/workflows/easy-cp.yml

```diff
@@ -0,0 +1,81 @@
+# Copyright 2023 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+name: Cherry-pick Labeled PR to Release Branch
+
+on:
+  pull_request:
+    branches: master
+    types: [labeled]
+
+permissions: write-all
+
+jobs:
+  cherrypick_to_release:
+    name: cherrypick_to_release
+    runs-on: ubuntu-latest
+    if: |
+      (github.event.label.name == format('cp{0} beta', ':') || github.event.label.name == format('cp{0} stable', ':')) &&
+      (github.event.pull_request.merged == true)
+    steps:
+      - name: Get Release Channel
+        run: |
+          echo "CHANNEL=$(echo ${{ github.event.label.name }} | cut -d ':' -f 2 | xargs)" >> $GITHUB_ENV
+      - name: Get Release Candidate Branch
+        run: |
+          curl -o version.json https://raw.githubusercontent.com/flutter/flutter/$CHANNEL/bin/internal/release-candidate-branch.version
+          echo "RELEASE_BRANCH=$(cat version.json | tr -d '\n')" >> $GITHUB_ENV
+          rm version.json
+      - name: Get Cherry Pick PR
+        run: |
+          echo "COMMIT_SHA=$(echo ${{ github.event.pull_request.merge_commit_sha }})" >> $GITHUB_ENV
+      - name: Checkout Flutter Repo
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11
+        with:
+          repository: flutter/flutter
+          token: ${{ github.token }}
+          path: flutter
+          ref: master
+          fetch-depth: 0
+      # use same name when checking out branch, since the marketplace action does a hard reset.
+      - name: Attempt CP
+        id: attempt-cp
+        working-directory: ./flutter
+        run: |
+          git config user.name "GitHub Actions Bot"
+          git config user.email "<>"
+          git fetch origin $RELEASE_BRANCH
+          git fetch origin master
+          git checkout -b cp-$COMMIT_SHA --track origin/$RELEASE_BRANCH
```

Comment:
Should we include the channel in the branch? I expect there's a good use case where people will want to open CPs at the same time to both beta and stable

Reply:
oh the branch name here actually has to be the same as $RELEASE_BRANCH. updated.

i.e., it has to be `git checkout -b $RELEASE_BRANCH --track origin/$RELEASE_BRANCH` cause the market place action does a hard reset on this name. 
The marketplace action has a step called `Create or update the pull request branch` that always does `/usr/bin/git reset --hard origin/$BRANCH_NAME`. if we use a fancy branch name other than $RELEASE_BRANCH, the market place action can't find it from remote and will throw an error.

Reply:
Is this something `branch-suffix` can help with?

Reply:
Good point, updated pushed branch name to support multiple CPs.

Yeah the branch names we checkout here in the container are local to our local checkout, and doesn't affect the branch name pushed to the release branch. later when we open the pull request, the pushed branch will have a different name. updated the name of the feature branch to be `cp-${{ env.CHANNEL }}-${{ env.COMMIT_SHA }}` via the `branch` field of the marketplace action.